autovizwidget/setup.py

```diff
@@ -59,7 +59,7 @@ def version(path):
       install_requires=[
           'plotly>=1.10.0,<1.11.0',
           'ipywidgets>5.0.0',
-          'hdijupyterutils==0.2.3',
+          'hdijupyterutils>=0.3',
```

Comment:
We might want to add < 0.4 in the future to make sure it doesn't automatically upgrade to the next incompatible major release.